fe/lib/components/SalaryDetails/SalaryDetails.tsx

```diff
@@ -87,6 +122,23 @@ export const SalaryDetails = ({
     min: validateMinAmount(min, errors),
   };
 
+  if (
+    payTypesLoading ||
+    currenciesLoading ||
+    payTypesData === undefined ||
+    currenciesData === undefined
```

Comment:
Do these need to explicitly test for `undefined`, or can we treat `null`s the same way?

Reply:
Realistically we should never get a `null` response with these 2 queries, if we did something went very wrong 😅 .

Reply:
Is the specific behaviour around `undefined` and not `null` guaranteed by the GraphQL server/client? If not I'd just err on the side of treating them the same in case someone decides to change the implementation in future.